dev-cpp/glibmm/glibmm-2.57.1.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="C++ interface for glib2"
+DESCRIPTION="A C++ API for parts of glib that are useful for C++."
+HOMEPAGE="https://www.gtkmm.org/en/"
+COPYRIGHT="2018 The gtkmm Development Team"
+LICENSE="GNU LGPL v2.1
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/glibmm/${portVersion%.*}/glibmm-$portVersion.tar.xz"
+CHECKSUM_SHA256="b98a9ea2ed5ea0a62d908182974e8a049fe7a890aad525b44a28bb5e44ca0d4c"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="1.3.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVers="2.58"
+
+PROVIDES="
+	glibmm$secondaryArchSuffix = $portVersion
+	lib:libgiomm$secondaryArchSuffix = $libVersionCompat
+	lib:libgiomm_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libglibmm$secondaryArchSuffix = $libVersionCompat
+	lib:libglibmm_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libglibmm_generate_extra_defs$secondaryArchSuffix = $libVersionCompat
+	lib:libglibmm_generate_extra_defs_$portVers$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libsigc_3.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	glibmm${secondaryArchSuffix}_devel = $portVersion
+	devel:libgiomm$secondaryArchSuffix = $libVersionCompat
+	devel:libgiomm_$portVers$secondaryArchSuffix = $libVersionCompat
+	devel:libglibmm$secondaryArchSuffix = $libVersionCompat
+	devel:libglibmm_$portVers$secondaryArchSuffix = $libVersionCompat
+	devel:libglibmm_generate_extra_defs$secondaryArchSuffix = $libVersionCompat
+	devel:libglibmm_generate_extra_defs_$portVers$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	glibmm$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libsigc_3.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:dot
+	cmd:doxygen
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:m4
+	cmd:make
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:xsltproc
+	"
+
+defineDebugInfoPackage glibmm$secondaryArchSuffix \
+	"$libDir"/libgiomm-$portVers.so.$libVersion \
+	"$libDir"/libglibmm-$portVers.so.$libVersion \
+	"$libDir"/libglibmm_generate_extra_defs-$portVers.so.$libVersion
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm -rf $libDir/*.la
+
+	mkdir -p $developDocDir
+	mv $dataDir/* $developDocDir
+	rmdir $dataDir
+
+	prepareInstalledDevelLibs libgiomm-$portVers \
+		libglibmm-$portVers libglibmm_generate_extra_defs-$portVers
+
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+		"$developDir" \
+		"$developDocDir" \
```

Comment:
As `developDocDir` resides within `developDir`, just specifing `developDir` should do the trick.

```diff
-		"$developDocDir" \
```